modal.css

```diff
@@ -0,0 +1,200 @@
+.modal {
+  display: none;
+}
+
+.modal.active {
+  display: block;
+  overflow-y: auto;
+  margin: 21px 6.5% auto 6.5%;
+  inset: 0;
+  border: 1px solid #fff;
+  gap: 24px;
+  position: fixed;
+  z-index: 777;
+  background: rgb(0, 0, 128);
+}
+
+#modal-header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-left: 6.5%;
+  margin-right: 6.5%;
+}
+
+#modal-header h1 {
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 28px;
+  margin-top: 38px;
+  display: flex;
+  align-items: center;
+  color: #fff;
+}
+
+.buttons button {
+  padding: 12px;
+  gap: 12px;
+  width: 108px;
+  height: 48px;
+  border-style: none;
+  background: #7d73fc;
+  box-shadow: 0 8px 16px rgba(125, 115, 252, 0.48);
+  display: flex;
+  color: #fff;
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 24px;
+}
+
+.buttons button:last-child {
+  margin-top: 24px;
+  margin-bottom: 28px;
+  padding: 12px;
+  gap: 12px;
+  width: 124px;
+  height: 48px;
+  background: #7d73fc;
+}
+
+#modal-header button {
+  margin-top: 24px;
+  cursor: pointer;
+  border: none;
+  outline: none;
+  font-size: 1.25rem;
+  color: #fff;
+}
+
+.modal-img {
+  box-sizing: border-box;
+  margin-left: 6.5%;
+  margin-right: 6.5%;
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+}
+
+.modal-img img:first-child {
+  width: 17.5rem;
+  height: 181px;
+}
+
+.modal-img img:nth-child(2) {
+  display: none;
+}
+
+.content p {
+  margin-left: 6.5%;
+  margin-right: 6.5%;
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  text-align: center;
+  color: #fff;
+  padding-left: 24px;
+  padding-right: 24px;
+}
+
+.content .tags {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 75%;
+  margin-left: 12.5%;
+}
+
+.content .tags li {
+  list-style: none;
+  font-family: "Poppins", sans-serif;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #fff;
+  border-right: 1px solid #fff;
+  padding-left: 8.5%;
+  padding-right: 8.5%;
+}
```

Comment:
- [ ] Your li tags have more spacing and kindly make the styling  that looks like Figma design and kindly fix the pop-up background color as well
### Your design
![his](https://user-images.githubusercontent.com/81727988/175774012-f84fd03d-d9f9-4084-a630-7751c2f98416.png)
### Figma design
![figma](https://user-images.githubusercontent.com/81727988/175774027-333b189c-7ff5-4b03-9c29-c8c9114358f2.png)